plugins/auth/index.js

```diff
@@ -83,13 +85,20 @@ exports.register = (server, options, next) => {
      * Generates a jwt that is signed and has a 2h lifespan
      * @method generateToken
      * @param  {Object} profile Object from generateProfile
+     * @param  {Integer} expires_sec JWT Expires time (must be seconds)
      * @return {String}         Signed jwt that includes that profile
      */
-    server.expose('generateToken', profile => jwt.sign(profile, pluginOptions.jwtPrivateKey, {
-        algorithm: ALGORITHM,
-        expiresIn: EXPIRES_IN,
-        jwtid: uuid()
-    }));
+    server.expose('generateToken', (profile, expiresSec = EXPIRES_IN) => {
+        const expiresIn = (expiresSec <= MAX_EXPIRES_IN)
+            ? expiresSec + EXPIRES_BUFFER
+            : MAX_EXPIRES_IN;
```

Comment:
We may as well plus the `EXPIRES_BUFFER` to the `MAX_EXPIRES_IN` value too.

Reply:
I see, I fixed!